platform-dom/src/main/java/com/softicar/platform/dom/elements/image/viewer/tag/DomImageViewerTagToggleButton.java

```diff
@@ -0,0 +1,18 @@
+package com.softicar.platform.dom.elements.image.viewer.tag;
+
+import com.softicar.platform.dom.DomI18n;
+import com.softicar.platform.dom.DomImages;
+import com.softicar.platform.dom.DomTestMarker;
+import com.softicar.platform.dom.elements.button.DomButton;
+import com.softicar.platform.dom.elements.image.viewer.DomImageViewer;
+
+public class DomImageViewerTagToggleButton extends DomButton {
```

Comment:
would be cool if this button would indicate its status somehow...if you don't have a quick idea then leave it as is

Reply:
I thought the same; however, I don't feel like sinking 2 hrs of time into Inkscape for this (due to my pathetic vector graphics skills). ;)

Reply:
I thought of something like opacity or filter: grayscale

Reply:
I think that would be confusing since we already use that effect to indicate "disabled" things.